entry-generation/godot-compiler-plugin/src/main/kotlin/godot/compilerplugin/CommonComponentRegistrar.kt

```diff
@@ -0,0 +1,59 @@
+package godot.compilerplugin
```

Comment:
not a fan of `compilerplugin` subpackage. How about `godot.kotlin.compiler`?

Reply:
agreed